Status Chart: Weekly updates

StephanTLavavej opened the pull request:
As usual, this updates dependencies, including es-module-shims 1.5.5 and luxon 2.4.0 for the browser.

:chart_with_downwards_trend: Live preview: [stephantlavavej.github.io/STL/](https://stephantlavavej.github.io/STL/)
===

strega-nil-ms commented:
```
PS C:\..\nimazzuc\projects\stl> pbgb
Enumerating objects: 13, done.
Counting objects: 100% (13/13), done.
Delta compression using up to 16 threads
Compressing objects: 100% (4/4), done.
Writing objects: 100% (5/5), 567 bytes | 567.00 KiB/s, done.
Total 5 (delta 3), reused 0 (delta 0), pack-reused 0
remote: Resolving deltas: 100% (3/3), completed with 3 local objects.
remote: This repository moved. Please use the new location:
remote:   https://github.com/StephanTLavavej/STL.git
remote: error: GH006: Protected branch update failed for refs/heads/gh-pages.
remote: error: This branch must not contain merge commits.
```

@StephanTLavavej your branch doesn't allow merge commits, so I can't push the merge with main; could you update yourself? (or is it okay to rebase here?)

Second question, is there a reason typescript hasn't been updated as well?

strega-nil-ms reviewed (approved):
I was informed that the merge conflict commit should be dropped instead.

Nit: update typescript.

StephanTLavavej commented:
@strega-nil-ms Thanks - I've updated TypeScript (with both `npm update` and `npm install [dependencies]@latest`), dropped the "DROP BEFORE MERGING" commit, and rebased on the current state of gh-pages. (I've verified that `npm run gather`, `npm run make`, and `npm run view` work normally, and we also have GitHub Actions verifying that TypeScript can compile both files.)
